mkdocs.yml

```diff
@@ -57,6 +57,8 @@ nav:
           - text-summarizer service: reference/services/text-summarizer.md
           - yolov8 service: reference/services/yolov8.md
           - ydata-profiling service: reference/services/ydata-profiling.md
+          - classification benchmark service: reference/services/classification-benchmark.md
```

Comment:
Try to add the menu entry in alphabetical order

Reply:
done